src/rust/ide/src/model/module.rs

```diff
@@ -236,6 +236,12 @@ impl QualifiedName {
     }
 }
 
+impl<'a> Into<&'a String> for &'a QualifiedName {
+    fn into(self) -> &'a String {
+        &self.0
+    }
+}
+
```

Comment:
can't we have rather something like AsRef<str> ?

Reply:
yep, AsRef would be better